src/Context/ProductContext.js

```diff
@@ -31,15 +48,15 @@ const ProductContext = ({children}) => {
     const filteredGender = filterGender(sortedData,gender);
     const filteredPrice = filterPrice(filteredGender, priceFilter);
     const filterData = filteredBrands(filteredPrice, brandsToBeFiltered);
-
+    const totalPrice = getTotalCartPrice(cart);
+    const discountPrice = getDiscountCartPrice(cart)
     return (
-    
-        <productContext.Provider value={{ dispatch, filterData }}>
+        <productContext.Provider value={{ dispatch, filterData, cart, wishlist, totalPrice, discountPrice }}>
             {children}
         </productContext.Provider>
     )
 }
 
 const useProducts = () => useContext(productContext);
 
-export { ProductContext, useProducts }
+export { ProductContext, useProducts }
```

Comment:
suggestion: name is misleading, its just a provider, so suffix Provider like ProductContextProvider